LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Trakx
+Copyright (c) 2022 Trakx
```

Comment:
[suggestion] Change to 2023.

Reply:
Thanks: https://github.com/trakx/kaiko-api-client/pull/53